spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminController.java

```diff
@@ -122,6 +125,38 @@
         chartColorsPerHttpResponseCode = Collections.unmodifiableMap(colorMap);
     }
 
+    private static final int oneMinuteToMilliseconds = 60000;
+    private static final int criticalThresholdInMinutes = 60;
```

Comment:
These values should be configurable, not hard-coded.

Reply:
Done